tasks/system_probe.py

```diff
@@ -846,6 +847,42 @@ def run_ninja(
         ctx.run(f"ninja {explain_opt} -f {nf_path} {target}")
 
 
+def setup_runtime_clang(ctx):
+    # check if correct version is already present
+    res = ctx.run("/opt/datadog-agent/embedded/bin/clang-bpf --version", warn=True)
+    if res.ok:
+        version_str = res.stdout.split("\n")[0].split(" ")[2].strip()
+        if version_str == CLANG_VERSION:
+            return
+
+    if not os.path.exists("/opt/datadog-agent/embedded/bin"):
+        if not is_root():
+            ctx.sudo("mkdir -p /opt/datadog-agent/embedded/bin")
+        else:
+            ctx.run("mkdir -p /opt/datadog-agent/embedded/bin")
+
+    arch = arch_mapping.get(platform.machine())
+    if arch == "x64":
+        arch = "amd64"
+
+    # download correct version from dd-agent-omnibus S3 bucket
+    clang_url = f"https://dd-agent-omnibus.s3.amazonaws.com/llvm/clang-{CLANG_VERSION}.{arch}"
+    if not is_root():
+        ctx.sudo(f"wget -q {clang_url} -O /opt/datadog-agent/embedded/bin/clang-bpf")
```

Comment:
`ctx.sudo` does not work as expected -- it requires the password to be supplied in a config file and not on the command line. Should just use `sudo` in front of the commands. See the note here: https://docs.pyinvoke.org/en/stable/api/context.html#invoke.context.Context.sudo

Reply:
Would sudo infront of the commands not have the same behaviour?

Reply:
It will prompt you for the password as expected, unlike `ctx.sudo` which shows the prompt but does not accept any input.